Formula/h/h5py.rb

```diff
@@ -0,0 +1,159 @@
+class H5py < Formula
+  include Language::Python::Virtualenv
+
+  desc "Read and write HDF5 files from Python"
+  homepage "https://www.h5py.org/"
+  url "https://files.pythonhosted.org/packages/37/fc/0b1825077a1c4c79a13984c59997e4b36702962df0bca420698f77b70b10/h5py-3.10.0.tar.gz"
+  sha256 "d93adc48ceeb33347eb24a634fb787efc7ae4644e6ea4ba733d099605045c049"
+  license "BSD-3-Clause"
+
+  depends_on "libcython" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python-pkgconfig" => :build
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.12" => [:build, :test]
```

Comment:
```suggestion
  depends_on "python@3.11" => [:build, :test]
  depends_on "python@3.12" => [:build, :test]
```

Reply:
This cannot be done, as `poetry` (which is dependency of `python-pkgconfig`) depends only on `python@3.12` and is not built for other Python versions.